polyphemus/bin/etl_tools

```diff
@@ -10,9 +10,10 @@ require 'yaml'
 config = YAML.load(File.read(File.expand_path("../../config.yml", __FILE__)))
 
 class RunOrTest < Etna::Command
+  attr_reader :runner
   def execute(script_file, record_id)
     runner = Polyphemus::MagmaEtlScriptRunner.new(script_file)
-    magma_client = Etna::Clients::Magma.new(token: ENV['TOKEN'], host: ENV['MAGMA_HOST'])
+    magma_client = Etna::Clients::Magma.new(token: ENV['TOKEN'], host: ENV['MAGMA_HOST'], persistent: false)
```

Comment:
So this makes things run slower ... but it seems to prevent the server from timing out long-lived connections which lead to issues like `/usr/local/lib/ruby/2.5.0/net/protocol.rb:189:in `rbuf_fill': end of file reached (EOFError)`.

Reply:
Yeah.  I have some suspicions about the ruby persistent connection / server side implementations unfortunately.  I've seen many unusual behaviors and thus I had begun turning it off.